### ISSUE TITLE
[Broadcom-DNX] Pause frames sent for Priority 0 traffic with new MMU buffer settings. 

### ISSUE THREAD

**Issue by amitpawar12** [sonic-net/sonic-buildimage#19646]
#### Description
With new MMU buffer settings ( #18239 ), there are Priority 0 PFC frames sent by DUT on congestion.

#### Steps to reproduce the issue:
1. Apply the new MMU buffer settings. Check the MMU configuration to ensure that buffer settings are applied.
```
Pool: ingress_lossless_pool
----  ----------
mode  dynamic
size  6441610000
type  both
xoff  396096307
----  ----------

Profile: egress_lossless_profile
----------  ---------------------
dynamic_th  -1
pool        ingress_lossless_pool
size        0
----------  ---------------------

Profile: egress_lossy_profile
----------  ---------------------
dynamic_th  -1
pool        ingress_lossless_pool
size        0
----------  ---------------------

Profile: ingress_lossy_profile
----------  ---------------------
dynamic_th  -6
pool        ingress_lossless_pool
size        0
xon_offset  0
----------  ---------------------

Profile: pg_lossless_100000_2000m_profile
----------  ---------------------
dynamic_th  -6
pool        ingress_lossless_pool
size        0
xoff        393216
xon         2560
xon_offset  363506
----------  ---------------------

```
2. Enable PFCWD and Credit-watchdog on the DUT.
3. Ensure that there are 2 ingress links and 1 egress link of the same link speeds.
4. Send traffic with lossless priorities 3 and 4 and lossy priorities 0, 1 and 2. Select equal ratio of traffic for all 5 priorities (20% in this case) through each of the ingress. One can also have each ingress send let's say 60% of link capacity oversubscribing the egress link.
5. DUT experiences congestion in this case due to oversubscription of the egress link.
6. Check for the PFCs for all the priorities.

Tagging @vmittal-msft for this issue.

#### Describe the results you received:
It is seen that DUT is sending PFCs for Priority 3 and 4 trying to stem the ingress lossless flows. However, it also sends PFCs for lossy priorities via Priority 0 Pause frames.

```
Received or Transmitted PFC counts 
ixre-egl-board73_ethernet0_tx_pfc_0 : 442314697 <<<<
ixre-egl-board73_ethernet0_tx_pfc_3 : 12588993 
ixre-egl-board73_ethernet0_tx_pfc_4 : 12528685 
ixre-egl-board73_ethernet8_tx_pfc_0 : 445819671  <<<<
ixre-egl-board73_ethernet8_tx_pfc_3 : 12593364 
ixre-egl-board73_ethernet8_tx_pfc_4 : 12524065
```

#### Describe the results you expected:
DUT should have sent PFCs for priority 3 and 4 only.

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by amitpawar12** [comment]
New MMU buffer settings are available as part of PR #19653.  We (nokia team) are working to see if the issue is seen again.

Thanks,
-A

**Comment by vmittal-msft** [comment]
This is fixed with latest MMU settings. Closing the issue.
